animations/src/main/java/com/chiralbehaviors/CoRE/meta/models/AttributeModelImpl.java

```diff
@@ -181,7 +181,7 @@ public String getJsonLdType(Attribute attribute) {
                                                            model.getKernel()
                                                                 .getJsonldType());
         if (type != null) {
-            return type.getTextValue();
+            return type.getValue();
```

Comment:
refactored all the type specific accessors from the attribute value, as it was a leaky abstraction.